src/routes/docs/products/ai/integrations/togetherai/+page.markdoc

```diff
@@ -0,0 +1,326 @@
+</head>
+```
+
+And after the `</head>` tag add a `<body>` which will contain the actual form:
+
+```html
+<body>
+  <main class="main-content">
+    <div class="top-cover u-padding-block-end-56">
+      <div class="container">
+        <div class="u-flex u-gap-16 u-flex-justify-center u-margin-block-start-16">
+          <h1 class="heading-level-1">Together AI Demo</h1>
+          <code class="u-un-break-text"></code>
+        </div>
+        <p class="body-text-1 u-normal u-margin-block-start-8" style="max-width: 50rem">
+          Use this page to test your implementation with Together AI. Enter
+          text and receive the model output as a response.
+        </p>
+      </div>
+    </div>
+    <div class="container u-margin-block-start-negative-56"
+      x-data="{ type: 'text', prompt: '', answer: {type: '', answer: ''}, loading: false }">
+      <div class="card u-flex u-gap-24 u-flex-vertical">
+        <div class="u-flex u-cross-center u-gap-8">
+          <div class="input-text-wrapper is-with-end-button u-width-full-line">
+            <input x-model="prompt" type="search" placeholder="Prompt" />
+            <div class="icon-search" aria-hidden="true"></div>
+          </div>
+          <div class="select u-width-140">
+            <select x-model="type">
+              <option value="text">Text</option>
+              <option value="image">Image</option>
+            </select>
+            <span class="icon-cheveron-down" aria-hidden="true"></span>
+          </div>
+
+          <button class="button" x-bind:disabled="loading"
+            x-on:click="async () => { loading = true; answer = {type: '', answer: ''}; try { answer = await onSubmit(prompt, type) } catch(err) { console.error(err); } finally { loading = false; } }">
+            <span class="text">Generate</span>
+          </button>
+        </div>
+        <template x-if="answer.type">
+          <div class="u-flex u-flex-vertical u-gap-12">
+            <div class="u-flex u-flex-vertical u-gap-12 card">
+              <div class="u-flex u-gap-12">
+                <h5 class="eyebrow-heading-2">Result:</h5>
+              </div>
+              <template x-if="answer.type === 'image'" class="u-flex u-gap-12">
+                <img class="u-max-width-400" x-bind:src="answer.response" alt="Together output" />
+              </template>
+              <template x-if="answer.type === 'text'" class="u-flex u-gap-12">
+                <p class="u-color-text-gray" x-text="answer.response"></p>
+              </template>
+            </div>
+          </div>
+        </template>
+      </div>
+    </div>
+  </main>
+</body>
+```
+
+Together, this will render a form to submit your question to the Appwrite Function through a POST request, which you'll create next. The Appwrite Function will call Together AI's API and return the response, which will be displayed on your page using different conditional statements depending on the output media type.
```

Comment:
```suggestion
Together, this will render a form to submit a query to the Appwrite Function through a `POST` request. The Appwrite Function invokes Together AI's API and returns the response, which will be displayed on the page using different conditional statements depending on the output media type.
```